src/Microsoft.AspNetCore.Server.Kestrel/Internal/Exceptions/BadHttpRequestException.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.IO;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Exceptions
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Exceptions
 {
     public sealed class BadHttpRequestException : IOException
```

Comment:
It seems weird to have a namespace with only one type. Any opposition to moving this to `Microsoft.AspNetCore.Server.Kestrel` directly?